.github/workflows/ibis-backends.yml

```diff
@@ -704,6 +704,70 @@ jobs:
         with:
           flags: backend,${{ matrix.backend.name }},${{ runner.os }},python-${{ steps.install_python.outputs.python-version }}
 
+  test_flink:
```

Comment:
Is a separate job necessary? The process seems relatively similar to tests for other backends, wonder if it can't be part of the same matrix.

Reply:
yes it's necessary bc python 11 is not supported yet. So I separate flink tests just like pyspark tests.

Reply:
You can just skip 3.11 in the matrix.

Reply:
beyond that reason, flink tests also required specific dependency (e.g. pandas 1.5) and environment variables. Therefore I think it's better to isolate this?

Reply:
after rebase your PR, I think it's better to merge the two jobs as you mentioned.